tests/integration/CMakeLists.txt

```diff
@@ -25,32 +25,34 @@
 # THE POSSIBILITY OF SUCH DAMAGE.
 
 
-set(COMMON_ARGS "--tstop 100. --celsius 6.3 -mpi")
+set(COMMON_ARGS "--mpi --tstop 100. config --celsius 6.3")
 set(RING_DATASET_DIR "${CMAKE_CURRENT_SOURCE_DIR}/ring")
-set(RING_COMMON_ARGS "--datpath ${RING_DATASET_DIR} ${COMMON_ARGS}")
-set(RING_GAP_COMMON_ARGS "--datpath ${CMAKE_CURRENT_SOURCE_DIR}/ring_gap ${COMMON_ARGS}")
-set(PERMUTE1_ARGS "--cell-permute 1 --nwarp 8")
-set(PERMUTE2_ARGS "--cell-permute 2 --nwarp 16")
+set(RING_COMMON_ARGS "${COMMON_ARGS} input --datpath ${RING_DATASET_DIR}")
+set(RING_SPIKEBUFFER_ARGS "${COMMON_ARGS} -b 1 input --datpath ${RING_DATASET_DIR}")
+set(RING_DUPLICATE_ARGS "${COMMON_ARGS} -z 3 input --datpath ${RING_DATASET_DIR}")
+set(RING_GAP_COMMON_ARGS "${COMMON_ARGS} input --datpath ${CMAKE_CURRENT_SOURCE_DIR}/ring_gap")
+set(PERMUTE1_ARGS "gpu --cell-permute 1 --nwarp 8")
+set(PERMUTE2_ARGS "gpu --cell-permute 2 --nwarp 16")
 if(ENABLE_OPENACC)
-  set(GPU_ARGS "--nwarp 8 --gpu")
+  set(GPU_ARGS "gpu --nwarp 8 --gpu")
```

Comment:
There seems to be some kind of issue with `--gpu` coming probably from here that fails the [jenkins_plan](https://bbpcode.epfl.ch/ci/blue/organizations/jenkins/oss.coreneuron.pipeline/detail/oss.coreneuron.pipeline/379/pipeline). We can check it together also at some point